cfme/cloud/instance/__init__.py

```diff
@@ -643,7 +643,8 @@ def prerequisite(self):
     def step(self, *args, **kwargs):
         # click the checkbox of every object in the filtered collection
         for entity in self.obj.all():
-            self.prerequisite_view.entities.get_entity(name=entity.name, surf_pages=True).check()
+            self.prerequisite_view.entities.get_entity(name=entity.name,
```

Comment:
This is covered by cfme/tests/cloud_infra_common/test_policy_simulation.py::test_policy_simulation_ui and will be added to PRT.